crates/router/src/configs/defaults.rs

```diff
@@ -5866,14 +5885,23 @@ impl Default for super::settings::RequiredFields {
                                             }
                                         ),
                                         (
-                                            "payment_method_data.bank_redirect.giropay.billing_details.billing_name".to_string(),
-                                            RequiredFieldInfo {
-                                                required_field: "payment_method_data.bank_redirect.giropay.billing_details.billing_name".to_string(),
-                                                display_name: "billing_name".to_string(),
-                                                field_type: enums::FieldType::UserBillingName,
-                                                value: None,
-                                            }
-                                        )
+                                                "billing.address.first_name".to_string(),
+                                                RequiredFieldInfo {
+                                                    required_field: "payment_method_data.billing.address.first_name".to_string(),
+                                                    display_name: "billing_name".to_string(),
+                                                    field_type: enums::FieldType::UserBillingName,
```

Comment:
field type has to be `UserFullName`

Reply:
We can use UserBillingName also, In sdk , value for `first_name` and `last_name` will be concatenated and will appear as one field Billing Name.